Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.h

```diff
@@ -106,11 +116,36 @@ class KeyboardMouse : public Core::Device
 
 private:
   void UpdateCursorInput();
+  void Generate_Octagon_Points(POINT octagon_points[8]);
+  bool Point_Is_Inside_Octagon(const POINT& mouse_point, const POINT octagon_points[8]);
+  double Calculate_Distance_between_Points(const POINT& first_point, const POINT& second_point);
+  long Find_Second_Line_Point(const POINT& mouse_point, const POINT octagon_points[8],
+                              long index_of_min_octagon_point);
+  void Move_Mouse_Point_Along_Gate(POINT& mouse_point, const POINT octagon_points[8]);
+  void Lock_Mouse_In_Jail(POINT& mouse_point);
+
+  bool player_requested_mouse_center = false;
+  const double screen_height = static_cast<double>(GetSystemMetrics(SM_CYFULLSCREEN));
+  const double screen_width = static_cast<double>(GetSystemMetrics(SM_CXFULLSCREEN));
+  const double screen_ratio = screen_width / screen_height;
+  const POINT center_of_screen =
+      POINT{static_cast<long>(screen_width / 2.0), static_cast<long>(screen_height / 2.0)};
+  enum octagon_points
```

Comment:
```suggestion
  enum OctagonPoint
```

Reply:
Is this a mandatory style change or just your personal preference? I was using it as a set of named variables so I was following variable naming conventions.

Reply:
I was also curious if enum class would have been more appropriate here. The compiler said to consider enum class but I couldn't figure out how to get them to work.

Reply:
Code style. Refer to [Contributing.md](/dolphin-emu/dolphin/blob/master/Contributing.md#naming) for more details.

Reply:
Huh, guess I just missed that. ty 👍